### PR TITLE
US120984 - Quiz Disable Right Click UI

### PR DESCRIPTION
**Pull request by wongvincent**
https://rally1.rallydev.com/#/detail/userstory/438844242444?fdp=true

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			d2l-input-checkbox {
+				padding-right: 1rem;
+			}
+
+			:host([dir="rtl"]) d2l-input-checkbox {
```
Comment (ronan-f) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js:27]: Not sure if we're concerned about duplicating these styles but we now have them in multiple editors

Comment (wongvincent) [reply components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-disable-right-click-editor.js:27]: yep..

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-hints-editor.js [hunk 2]
```diff
@@ -48,10 +45,6 @@ class ActivityQuizHintsEditor
 		}
 
 		return html`
-			<label class="d2l-label-text">
-				${this.localize('displayTools')}
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-hints-editor.js:52]: Was this leftover from some other use?

File: components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-timing-and-display-editor.js [hunk 3]
```diff
@@ -47,7 +50,11 @@ class ActivityQuizTimingAndDisplayEditor extends AsyncContainerMixin(LocalizeAct
 				<li slot="summary-items">${this._renderAllowHintsSummary()}</li>
 
 				<div class="d2l-editors" slot="components">
+					<label class="d2l-label-text">
+						${this.localize('displayTools')}
```
Comment (mostlyfabulous) [inline components/d2l-activity-editor/d2l-activity-quiz-editor/d2l-activity-quiz-timing-and-display-editor.js:54]: Oh it moved here

**Review by mdulat — APPROVED** [review]

**Pull request merged**